com.milesalan.mepo.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id>com.milesalan.mepo</id>
+  <metadata_license>GPL</metadata_license>
```

Comment:
this metadata license isn't acceptable (it is JUST for this file). `CC0-1.0` is prefered.

you are missing the project license.

Reply:
Fixed